### DISCUSSION TITLE
Wrong links in the "Welcome" section

### DISCUSSION THREAD

**Discussion by razvanilin** [paircodedev/sandbox#13] [Bug Report] [answered]
The titles marked below are leading to external websites (probably placeholders)

![Screenshot 2021-06-01 at 10 32 42](https://user-images.githubusercontent.com/5342321/120258601-eec1f800-c2c4-11eb-85b7-f68ccf1f0880.jpg)

**Comment by Necmttn** [comment, marked as answer]
Hey @razvanilin, 
Thank you for catching this bug. Indeed they were placeholder links from the boilerplate 🙇‍♂️ 

Cheers ! 🍻
